src/coreclr/src/tools/crossgen2/crossgen2/Properties/Resources.resx

```diff
@@ -138,6 +138,9 @@
   <data name="InputBubbleOption" xml:space="preserve">
     <value>True when the entire input forms a version bubble (default = per-assembly bubble)</value>
   </data>
+  <data name="CompileNoMethodsOption" xml:space="preserve">
+    <value>True to not compile any methods into the R2R image (default = false)</value>
+  </data>
```

Comment:
maybe: `True to skip compiling methods into the R2R image`?

Reply:
Again, I should never be allowed to write customer visible text.    :) I like your wording better.